.github/workflows/pwa-testnet-master.yml

```diff
@@ -3,7 +3,7 @@ name: PWA Testnet Workflow
 on:
   push:
     branches:
-      - dev
+      - master
```

Comment:
Can we use a single workflow for botch branches?

Reply:
https://github.com/Adamant-im/adamant-im/pull/773/commits/f6e6a2fb94751cda65f8ba5fda49f1246f8af3e8